package-lock.json

```diff
@@ -29,7 +29,7 @@
         "@hashicorp/platform-cms": "^0.1.0",
         "@hashicorp/platform-code-highlighting": "0.1.1",
         "@hashicorp/platform-docs-mdx": "0.1.1",
-        "@hashicorp/platform-nextjs-plugin": "0.2.0",
+        "@hashicorp/platform-nextjs-plugin": "1.0.1",
```

Comment:
🤔 🤔 🤔 I merely did the usual `npm i && npm start` to make sure I was up-to-date before running the local server. I wouldn't think this package would upgrade unless I told it to, given the version is pinned? 

Does anyone reading this have any thoughts? After brushing up on the [versioning page in the Node docs](https://nodejs.dev/learn/semantic-versioning-using-npm/) it seems this shouldn't have happened 🤔

Reply:
Ah, could be related to this issue:
https://github.com/npm/npm/issues/17507

Might have to take another look at my `node` and `npm` versions.